.gitignore

```diff
@@ -43,3 +43,4 @@ Gemfile.lock
 venv
 bears/upload/
 rultor_secrets.sh
+.pytest_cache
```

Comment:
Add "Closes https://github.com/coala/coala-bears/issues/2326" to your second commit. Also I recommend being asked to be assigned to that issue or dropping this change because a lot of newcomers want to be assigned to the issue